kmod/src/server.c

```diff
@@ -3989,6 +3990,15 @@ void scoutfs_server_abort(struct super_block *sb)
 	stop_server(server);
 }
 
+void scoutfs_server_abort_conn(struct super_block *sb)
+{
+	struct server_info *server = SCOUTFS_SB(sb)->server_info;
+
+	if (server->conn) {
```

Comment:
It turns out this is dangerous because the server conn lifetime is different than the client.  For the client, we have a conn for the duration of the mount so it's a lot easier to reference the conn from within the mount.   For the server, the conn is allcoated and freed each time the server starts and stops.   It'd be pretty easy for this to deref a null conn.   It turns out all we should do is just call scoutfs_server_abort() from umount_begin, we don't need this _server_abort_conn at all, I don't think.

Reply:
Okay this makes sense now since I was trying to avoid the deref of a null in the above with:
`while (!server->shutting_down && !scoutfs_forcing_unmount(sb)) {`